_data/people/fkw888.yml

```diff
@@ -6,6 +6,12 @@ institution: University of California, San Diego
 website: https://www-physics.ucsd.edu/fac_staff/fac_profile/faculty_description.php?person_id=494
 photo: /assets/images/team/Frank-Wuerthwein.jpg
 presentations:
+  - Title: "Advancing Open Science through distributed High Throughput Computing"
```

Comment:
Does the system that processes this data treat `Title` the same as `title`? I would probably change this label to lowercase, just to be sure.

Reply:
Fixed